src/common.ts

```diff
@@ -49,7 +49,9 @@ export class PushOnlyObserver<A> implements BListener, SListener<A> {
     }
   }
   pushB(t: number): void {
-    this.callback((this.source as any).last);
+    const b = <Behavior<A>>this.source;
+    b.pull(t);
+    this.callback(b.last);
```

Comment:
Why this change adding a `pull`? The tests pass without this change?

Reply:
this is some remains of my first attempt, where the pull was not trigger during the traversal of push. I will remove this again :)